python3-aeneas.yml

```diff
@@ -0,0 +1,99 @@
+# NOTES:
+  # 1.7.3.0 is the latest aeneas version on pypi. Are there newer versions
+  # and sources elsewhere? 
+  # This file uses version 1.25.2 of numpy which worked without having to
+  # install extra build modules. The latest version of numpy is 2.0.0 which
+  # will require 'meson-python' to build successfully.
+  # A setup error in version 1.17.5 of numpy suggested the "setup.py clean"
+  # function was deprecated for builds. This is why I was testing new versions.
+  # FYI: The last version of numpy released as a .zip instead of a .tar.gz
+  # was 1.22.4: https://pypi.org/project/numpy/1.22.4/#files
+name: python3-aeneas
+buildsystem: simple
+build-options:
+    append-ld-library-path: /app/lib
+build-commands:
+    - echo PWD="${PWD}"  and  FLATPAK_DEST="${FLATPAK_DEST}"
+    - python3 -m pip install --upgrade pip
+    # Ensuring python3 points to the right place if needed
+    #- rm -f /app/bin/python
+    - python3 -m pip list
+    - python3 -m pip install --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "numpy" --no-build-isolation
+    - python3 -m pip install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "aeneas" --no-build-isolation
+sources:
+    # Latest (on pypi at least) : 1.7.3.0
+    - type: file
+      url: https://files.pythonhosted.org/packages/e5/92/88d421001bb257588df4864ceca24d570e2e822db4f40f48737a78b648b2/aeneas-1.7.3.0.tar.gz
+      sha256: b492d8ba36f12cce69c29c35e7075857a9d036c4743cc8f4689922be980826da
+    
+    # Latest: 2.5
+    - type: file
+      url: https://files.pythonhosted.org/packages/4c/f3/038b302fdfbe3be7da016777069f26ceefe11a681055ea1f7817546508e3/soupsieve-2.5-py3-none-any.whl
+      sha256: eaa337ff55a1579b6549dc679565eac1e3d000563bcb1c8ab0d0fefbc0c2cdc7
+    
+    #OLD
```

Comment:
Can you remove the commented out sources?

Reply:
Removed